### DISCUSSION TITLE
NPM official api and recently updated packages

### DISCUSSION THREAD

**Discussion by Yao-Wen-Chang** [community/community#132191] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hey guys,

Currently, I am doing cybersecurity research.
I need the data of the latest updated packages (within one day). 
I found this endpoint: https://www.npmjs.com/browse/updated; however, I am wondering if there
are parameters to limit the results.

**Comment by SajanGhimire1** [comment]
@Yao-Wen-Chang 
The NPM API doesn’t support direct filtering by update date. Instead, use ```https://registry.npmjs.org/-/v1/search``` to fetch a broad dataset and programmatically filter packages by their date attribute for recent updates.

**Reply by Yao-Wen-Chang** [reply]
I see. Thank you so much.
